### DISCUSSION TITLE
Voting for new plugins

### DISCUSSION THREAD

**Discussion by Darginec05** [yoopta-editor/Yoopta-Editor#373] [Polls]
Table was chosen in [previous poll](https://github.com/Darginec05/Yoopta-Editor/discussions/197).
It's time to choose the next plugin to develop 🚀

**Comment by Akash-tech-org** [comment]
Hey @Darginec05 do you have some plans for live collaboration feature ? because I'm building an app where I need live collaboration.

**Comment by Darginec05** [comment]
@Akash-tech-org it is currently under development

**Reply by Akash-tech-org** [reply]
Hey @Darginec05 as I'm creating a roadmap/idea for my project. May I know when will be the live collab feature will be available ? Thanks.

**Reply by Darginec05** [reply]
@Akash-tech-org I think it will be ready for use in month

**Reply by Akash-tech-org** [reply]
Great thanks.

**Comment by crow-rojas** [comment]
Hi @Darginec05! Are you planning to add LaTeX (with KaTeX maybe) support in the future? I would love to help if is in the roadmap:)

**Reply by Darginec05** [reply]
Sure! This is one of the plugins that the community is really looking forward to. :) 
We already have opened issue with this - https://github.com/yoopta-editor/Yoopta-Editor/issues/275

It would be nice if you could help with the implementation :)
Could you get started?

**Reply by crow-rojas** [reply]
@Darginec05 Good one! I think I can see how to start implementing the LaTeX functionality on my weekends, or in my week evenings, since I have my full time job. Do you know if anyone already started working on the implementation? Where do you recommend me to start?
